tests/Integration/PrestaShopBundle/Form/EventListener/ProductTypeListenerTest.php

```diff
@@ -58,13 +58,19 @@ public function testSubscribedEvents(): void
     public function testFormTypeExistsInFormDependingOnProductType(string $productType, string $formTypeName, bool $shouldExist): void
     {
         $form = $this->createForm(TestProductFormType::class);
+
         $this->assertFormTypeExistsInForm($form, $formTypeName, true);
         $this->adaptProductFormBasedOnProductType($form, $productType);
         $this->assertFormTypeExistsInForm($form, $formTypeName, $shouldExist);
     }
 
     public function getFormTypeExpectationsBasedOnProductType(): Generator
     {
+        yield [ProductType::TYPE_PACK, 'description.packed_products', true];
+        yield [ProductType::TYPE_STANDARD, 'description.packed_products', false];
+        yield [ProductType::TYPE_COMBINATIONS, 'description.packed_products', false];
+        yield [ProductType::TYPE_VIRTUAL, 'description.packed_products', false];
```

Comment:
Just a suggestion: you can name each test case to make test failures easier to locate
```suggestion
        yield 'pack type' => [ProductType::TYPE_PACK, 'description.packed_products', true];
        yield 'standard type' => [ProductType::TYPE_STANDARD, 'description.packed_products', false];
        yield 'combination type' => [ProductType::TYPE_COMBINATIONS, 'description.packed_products', false];
        yield 'virtual type' => [ProductType::TYPE_VIRTUAL, 'description.packed_products', false];
```

Reply:
I didn't know that we are able to do that ! Thx for the tip !